rasa/nlu/extractors/crf_entity_extractor.py

```diff
@@ -121,20 +83,69 @@ def required_components(cls) -> List[Type[Component]]:
         "digit": lambda crf_token: crf_token.text.isdigit(),
         "pattern": lambda crf_token: crf_token.pattern,
         "text_dense_features": (
-            lambda crf_token: CRFEntityExtractor._convert_dense_features_for_crfsuite(
+            lambda crf_token: CRFEntityExtractorGraphComponent._convert_dense_features_for_crfsuite(  # noqa: E501
                 crf_token
             )
         ),
         "entity": lambda crf_token: crf_token.entity_tag,
     }
 
+    @staticmethod
+    def get_default_config() -> Dict[Text, Any]:
+        """The component's default config (see parent class for full docstring)."""
+        return {
+            # BILOU_flag determines whether to use BILOU tagging or not.
+            # More rigorous however requires more examples per entity
+            # rule of thumb: use only if more than 100 egs. per entity
+            BILOU_FLAG: True,
+            # Split entities by comma, this makes sense e.g. for a list of ingredients
+            # in a recipie, but it doesn't make sense for the parts of an address
+            SPLIT_ENTITIES_BY_COMMA: True,
+            # crf_features is [before, token, after] array with before, token,
+            # after holding keys about which features to use for each token,
+            # for example, 'title' in array before will have the feature
+            # "is the preceding token in title case?"
+            # POS features require SpacyTokenizer
+            # pattern feature require RegexFeaturizer
+            "features": [
+                ["low", "title", "upper"],
+                [
+                    "low",
+                    "bias",
+                    "prefix5",
+                    "prefix2",
+                    "suffix5",
+                    "suffix3",
+                    "suffix2",
+                    "upper",
+                    "title",
+                    "digit",
+                    "pattern",
+                ],
+                ["low", "title", "upper"],
+            ],
+            # The maximum number of iterations for optimization algorithms.
+            "max_iterations": 50,
+            # weight of the L1 regularization
+            "L1_c": 0.1,
+            # weight of the L2 regularization
+            "L2_c": 0.1,
+            # Name of dense featurizers to use.
+            # If list is empty all available dense features are used.
+            "featurizers": [],
+        }
+
     def __init__(
         self,
-        component_config: Optional[Dict[Text, Any]] = None,
+        config: Dict[Text, Any],
+        model_storage: ModelStorage,
+        resource: Resource,
         entity_taggers: Optional[Dict[Text, "CRF"]] = None,
     ) -> None:
-
-        super().__init__(component_config)
+        """Declare instance variables with default values."""
```

Comment:
I don't understand the docstring: which default values?

Reply:
Ah I copied this over from another component, I'll change the docstring instead.